Sources/Misc/OperationDispatcher.swift

```diff
@@ -20,6 +20,8 @@ class OperationDispatcher {
     private let workerQueue = DispatchQueue(label: "OperationDispatcherWorkerQueue")
     private let maxJitterInSeconds: Double = 5
 
+    static let main: OperationDispatcher = .init()
```

Comment:
The only reason the methods in this class aren't `static` is so we can mock it, but there's no reason why we can't share it.